tests/magics/tcompilesetting.nim

```diff
@@ -0,0 +1,19 @@
+discard """
```

Comment:
should be:
tcompilesettings.nim 
not compilesetting.nim 
for consistency with std/compilesettings.nim